blog/2015-09-30-HowLuckyAreYourRandomSeeds.adoc

```diff
@@ -0,0 +1,141 @@
+= How lucky are your random seeds?
+:page-interpolate: true
+:awestruct-author: oskopek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [production]
+
+For a long time, it has been speculated that OptaPlanner performs differently when it's random number generator
```

Comment:
Fixed.

I never assumed RNG == random seed, but the fact you think so by reading the paragraph means I wrote it badly :). Hopefully fixed.